pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -867,6 +867,10 @@ type Flux struct {
 	// +optional
 	Namespace string `json:"namespace,omitempty"`
 
+	// Path to the kubernetes config for the cluster
+	// +optional
+	Kubeconfig string `json:"kubeconfig,omitempty"`
+
```

Comment:
worth noting the behaviour when not set?

Reply:
maybe? the behaviour isn't ours, in our case we just don't do anything, but i can note flux's default